content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -132,7 +132,9 @@ stages:
       - performance.json
   training:
     desc: train your model
-    cmd: python train.py
+    cmd:
+      - test -z "$(git status -s)"
+      - python train.py
```

Comment:
> test -z "$(git status -s)"

I'd maybe use something much simpler so no one gets confused by this. Maybe

```suggestion
      - echo "Train my model!"
      - python train.py
```

Reply:
indeed I had to look at the command on google! But I was looking for a relevant use case, echoing while the command is printer as well seems a bit pointless. Maybe `git pull` ?

Reply:
Yeah I know my example goes too far into the useless side ha, sorry. `git pull` is probably not something you would do inside DVC stages, as Git manages the dvc.yaml file in the first place. `echo` is fine here if we can't come up with a better (simple) idea.

Reply:
> if we can't come up with a better (simple) idea

Are there examples somewhere in the core repo where this feature was originally proposed perhaps? I mean, what's the main use case for multiple commands in the first place?

Reply:
well, nothing mentionne in the issue at least

Reply:
maybe just the `git status` then?